haec/prisma/schema.prisma

```diff
@@ -14,3 +14,20 @@ datasource db {
 }
 
 
+model User {
+  id    Int     @default(autoincrement()) @id
+  email String  @unique
+  name  String?
+
+  @@map("users")
+}
+
+model horses {
+  id  Int @default(autoincrement()) @id
+  show_name  String @unique
```

Comment:
Likewise for field names, these should be `camelCase` with a mapping to fit the SQL best practices.

For example this should be:
```suggestion
showName    String   @unique     @map("show_name")
```

So in combination with the above table mapping, this would make the sql:
```sql
select show_name from horses;
```

Reply:
Likewise this change for the rest of the fields 👇